### PR TITLE
Add support for ResizeMode

### PR DESCRIPTION
**Pull request by KangYee**
Hi, I really like this repository, but I noticed that the ResizeMode is currently unavailable, I hope this PR can be helpful.

### CODE REVIEW

File: compose-video/src/main/kotlin/io/sanghun/compose/video/ResizeMode.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+/**
+ * Convert [ResizeMode] to playerview resize mode.
+ *
+ * @return [AspectRatioFrameLayout.RESIZE_MODE_FIT] or [AspectRatioFrameLayout.RESIZE_MODE_FIXED_WIDTH]
+ * or [AspectRatioFrameLayout.RESIZE_MODE_FIXED_HEIGHT] or [AspectRatioFrameLayout.RESIZE_MODE_FILL]
+ * or [AspectRatioFrameLayout.RESIZE_MODE_ZOOM]
+ */
+@OptIn(UnstableApi::class)
+internal fun ResizeMode.toPlayerViewRepeatMode(): Int =
```
Comment (dsa28s) [inline compose-video/src/main/kotlin/io/sanghun/compose/video/ResizeMode.kt:63]:
```suggestion
internal fun ResizeMode.toPlayerViewResizeMode(): Int =
```

**Review by dsa28s — COMMENTED** [review]
Thank you for your contribution 👍 I left a comment, so please review it.

File: compose-video/src/main/kotlin/io/sanghun/compose/video/ResizeMode.kt (hunk 1 above)
Comment (KangYee) [reply compose-video/src/main/kotlin/io/sanghun/compose/video/ResizeMode.kt]: My god, I forgot about this, thanks, I have resubmitted the code, please review it again.

**Review by dsa28s — APPROVED** [review]
LGTM 👍

**Comment by dsa28s** [comment]
I will release it as 1.3.0 along with other features.
Thanks for your contribution!

**Comment by KangYee** [comment]
Wow! This is good news, thanks.

**Pull request merged**